proto/protovalidate-testing/buf/validate/conformance/cases/custom_constraints/custom_constraints.proto

```diff
@@ -70,48 +70,6 @@ message MessageExpressions {
   }
 }
 
-message FieldExpressions {
```

Comment:
This message is replaced by several individual messages. Makes the test cases easier to follow. Makes it easier to add more tests.